lib/c.c

```diff
@@ -677,9 +686,15 @@ void free(void *ptr)
     if (!ptr)
         return;
 
+    /* TODO: it takes long time to search in chuncks */
```

Comment:
FYI. An excellent video: [malloc sucks so i wrote my own heap in C](https://youtu.be/CulF4YQt6zA).